.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+\<Remember to add meaningful title\>
+
+\<Short description of the PR\>
+
+Fixes #\<issue number\>
+
+Before merge following needs to be applied:
+- [ ] At least one example testcase added in testcases/
+- [ ] Library Documentation regenerated according to [Generate docs](https://github.com/devopsspiral/KubeLibrary#development)
```

Comment:
I would recommend to make "Generate docs" on the readme a H3 to be able to directly link to the anchor e.g. https://github.com/devopsspiral/KubeLibrary#generate-docs